modules/features/discover/src/main/res/layout/fragment_discover.xml

```diff
@@ -1,45 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm not sure how the Discover page change got in here. I will focus on this section in the next PR.